nixos/modules/tasks/network-interfaces.nix

```diff
@@ -483,7 +483,6 @@ in
     };
 
     networking.fqdn = mkOption {
-      readOnly = true;
       type = types.str;
       default = if (cfg.hostName != "" && cfg.domain != null)
         then "${cfg.hostName}.${cfg.domain}"
```

Comment:
Can you change the error message below please? After this pr `fqdn` can be set explizitly and is not always derived from hostname and domain name

Reply:
Done 👍